docs_app/content/tutorial/basics.md

```diff
@@ -0,0 +1,176 @@
+# The basics
+
+## Converting to observables
+
+<!-- skip-example -->
```

Comment:
I think the skip-example comments were used by the old docs. Nevertheless it don't have an impact in the new docs, therefore I would remove it